zephyr/wrapper.c

```diff
@@ -28,6 +28,13 @@
 #error Define CONFIG_DYNAMIC_INTERRUPTS
 #endif
 
+
+/* TODO bring back zephyr MAX implementation which is used by the logging in
+ * that file.
+ */
+#undef MAX
```

Comment:
Why not `undef` it before the Zephyr includes section (line 21) and then get Zephyr's `MAX` via an include?